docs/en/security/host-based-security-workflows.rst

```diff
@@ -8,17 +8,17 @@ Host Based Security Workflows
 Introduction
 ------------
 
-It is recommended to have an uninterrupted power supply while enabling security features on ESP32 SoCs. Power failures during secure manufacturing process could cause issues that are hard to debug and, in some cases, may cause permanent boot-up failures.
+It is recommended to have an uninterrupted power supply while enabling security features on ESP32 SoCs. Power failures during the process of secure boot enabling could cause issues that are hard to debug and, in some cases, may cause permanent boot-up failures.
```

Comment:
the document is generic and related to manufacturing processes. Secure boot is a part of that so I think the previous line should be fine. Other than that the MR looks really good to me, Thanks for fixing the documentation

Reply:
Thanks for your feedback. I have updated the document according to your comment.